### PR TITLE
DriverFactory and Listners class addition

### PR DESCRIPTION
**Pull request by Subha6636**
I have added Driver Factory and Listeners to take failed screenshot.

### CODE REVIEW

File: src/test/java/com/taspro/base/DriverFactory.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	private DriverFactory() {
+
+	}
+
+	public static DriverFactory getDriverFactoryInstance() {
+		return instance;
+	}
+
+	public WebDriver initializeDriver() {
```
Comment (bikash9090) [inline src/test/java/com/taspro/base/DriverFactory.java:27]: Add string parameter to get the browser name .

File: src/test/java/com/taspro/listeners/ReportListners.java
Comment (bikash9090) [inline src/test/java/com/taspro/listeners/ReportListners.java:1]: remove not in use imports.

File: testng.xml
Comment (bikash9090) [inline testng.xml:1]: rearrenge the code.

**Review by bikash9090 — CHANGES_REQUESTED** [review]

File: src/test/java/com/taspro/base/DriverFactory.java (hunk 1 above)
Comment (bikash9090) [inline src/test/java/com/taspro/base/DriverFactory.java]: add string parameter to get the browser name.

File: src/test/java/com/taspro/base/DriverFactory.java
Comment (bikash9090) [inline src/test/java/com/taspro/base/DriverFactory.java:1]: Remove the reproperties from here and it will be taken care by the caller class.

**Review by bikash9090 — CHANGES_REQUESTED** [review]

File: src/test/java/com/taspro/base/DriverFactory.java (hunk 1 above)
Comment (Subha6636) [reply src/test/java/com/taspro/base/DriverFactory.java]: Done.

File: src/test/java/com/taspro/base/DriverFactory.java
Comment (Subha6636) [reply src/test/java/com/taspro/base/DriverFactory.java:1]: Done.

**Review by bikash9090 — APPROVED** [review]
Approved for merge into branch.

**Pull request merged**